installer/README.md

```diff
@@ -53,7 +53,7 @@ dependencies installed correctly.
   - boto3 (1.9)
   - gitpython
 
-  3. Install the Terraform from https://releases.hashicorp.com/terraform/0.11.15/terraform_0.11.15_linux_amd64.zip
+  3. Install the latest version of Terraform from https://learn.hashicorp.com/terraform/getting-started/install.html
```

Comment:
We are maintaining the main installer steps in the wiki. That is the reason why these steps may be out dated. We need to duplicate the wiki installer here to avoid confusion.